src/wasm/tf-builder.cc

```diff
@@ -569,6 +579,51 @@ void TFBuilder::ReturnVoid() {
 }
 
 
+TFNode* TFBuilder::MakeI32Popcnt(TFNode* input) {
```

Comment:
Please drop in a TODO to factor this code out into a stub, since in the long run we don't want to inline this every time.